src/shopify/theme_client.go

```diff
@@ -142,7 +142,7 @@ func (c Client) GetShop() (Shop, error) {
 
 // Themes will return all the available themes on a domain.
 func (c Client) Themes() ([]Theme, error) {
-	resp, err := c.http.Get(APIPath + "themes.json", nil)
+	resp, err := c.http.Get(APIPath+"themes.json", nil)
```

Comment:
Did `gofmt` change some of these?

Reply:
yup

Reply:
I opened https://github.com/Shopify/themekit/pull/827 to keep the noise out of this one.